src/clients/vendor.rb

```diff
@@ -98,18 +98,15 @@ def main
       else
         Pkg.TargetInit("/", false)
 
-        @products = Pkg.ResolvableProperties("", :product, "")
+        @products = Y2Packager::Resolvable.find(kind: :product,
+           status:    :installed,
+           category:  "base")
         @products = [] if @products == nil
-        @products = Builtins.filter(@products) do |p|
-          Ops.get(p, "status") == :installed
-        end
-        @base = Builtins.filter(@products) do |p|
-          Ops.get_string(p, "category", "") == "base"
-        end
         @base = deep_copy(@products) if Builtins.size(@base) == 0
-        @product = Ops.get(@base, 0, {})
-        @version = Ops.get_string(@product, "version", "")
-        @version = Ops.get(Builtins.splitstring(@version, "-"), 0, "") # split off release
+        @product = @base[0]
+        if @product
+          @version = @product.version_version
```

Comment:
Admittedly I had to check pkg-bindings because that `version_version` method name looks wrong to me. But it is right :smile:. The version is composed of a `version_version`, a `version_release`, and a `version_epoch`.